test/lib/code_corps/model/task_list_test.exs

```diff
@@ -16,12 +17,122 @@ defmodule CodeCorps.TaskListTest do
     refute changeset.valid?
   end
 
-  test "is not inbox by default" do
+  test "defaults :done to 'false'" do
     {:ok, record} =
-      %TaskList{}
-      |> TaskList.changeset(@valid_attrs)
-      |> CodeCorps.Repo.insert
+      %TaskList{} |> TaskList.changeset(@valid_attrs) |> Repo.insert
+    assert record.done == false
+  end
+
+  test "defaults :inbox to 'false'" do
+    {:ok, record} =
+      %TaskList{} |> TaskList.changeset(@valid_attrs) |> Repo.insert
+    assert record.inbox == false
+  end
+
+  test "defaults :pull_requests to 'false'" do
+    {:ok, record} =
+      %TaskList{} |> TaskList.changeset(@valid_attrs) |> Repo.insert
+    assert record.pull_requests == false
```

Comment:
@snewcomer Your assertion here was `refute record.pull_requests`, but that one would have passed even if `record.pull_requests == nil`, so we want a more explicit one.

Reply:
:man_facepalming: